tools/importer-rest-api-specs/components/parser/temp_removed_after_sdk_migration.go

```diff
@@ -0,0 +1,80 @@
+package parser
```

Comment:
This file exists only temporarily/until the SDK migration is completed - it just means we can reuse these helper functions early